ci-operator/jobs/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-release-1.8-presubmits.yaml

```diff
@@ -59,17 +59,17 @@ presubmits:
   - agent: kubernetes
     always_run: true
     branches:
-    - ^main$
-    - ^main-
-    cluster: build09
+    - ^release-1\.8$
+    - ^release-1\.8-
+    cluster: build03
```

Comment:
Is it required to provide the cluster name? Will it be available always?

Reply:
This is the effect of running `make jobs` as suggested in failed CI output with previous versions of this PR... I honestly don't know what this means :disappointed:

Reply:
Ok, I was also not sure. Reading a bit on the CI clusters, it looks like the list of available CI clusters comes from here - https://docs.ci.openshift.org/docs/getting-started/useful-links/#clusters
The CI job needs OCP version 4.16 and build03 is 4.16. build09 is 4.17
Can you add the clusters link as a comment in the job yaml? It'll be useful going forward :-)

Reply:
> Ok, I was also not sure. Reading a bit on the CI clusters, it looks like the list of available CI clusters comes from here - https://docs.ci.openshift.org/docs/getting-started/useful-links/#clusters The CI job needs OCP version 4.16 and build03 is 4.16. build09 is 4.17 Can you add the clusters link as a comment in the job yaml? It'll be useful going forward :-)

I gave a try but the comment gets reverted by `make jobs` :wink: